doc/syscalls/60009_distance.md

```diff
@@ -0,0 +1,61 @@
+---
+driver number: 0x60009
+---
+
+# Distance
+
+## Overview
+
+The distance sensor driver allows a process to read the distance measured by a HC-SR04 distance sensor. Distance is reported in millimeters.
```

Comment:
```suggestion
The distance sensor driver allows a process to read the distance measured by a distance sensor. Distance is reported in millimeters.
```

I think this interface looks like it aims to be generic across any distance sensor. If for some reason it should be specific to that exact sensor, it should be in the `0x7____` family of drivers.

Reply:
I'd agree that the distance interface is generic. So the one-word change here seems appropriate.